cmd/zdb/resource.h

```diff
@@ -0,0 +1,15 @@
+//{{NO_DEPENDENCIES}}
```

Comment:
Could we perhaps move the files `resource.h` and `resource.rc` into `os/windows/` for `cmd/zdb` `cmd/zfs` and `cmd/zstream` like that of `cmd/zpool` ?

Reply:
Yes, I have amend the changes.